markdown/guide/creating-an-engine.md

```diff
@@ -212,5 +212,9 @@ export default buildRoutes(function() {
   });
 });
 ```
+To create a route within an in-repo-engine, you can run
```

Comment:
can you add a topic about `generate` for all the things into `in-repo-engine`? with this way, we can make things more explicit

Reply:
Sure thing, will update PR soon